src/movies.html

```diff
@@ -18,6 +18,8 @@
                 </a>
             </div>
             <div class="header__links">
+                <a href="./index.html">
+                <img class="log-out" src="./images/films/log of.svg" alt="Logo"></a>
```

Comment:
maybe detele this image. these picture will need to be added in the basic styles.